src/Bridge/Doctrine/Persister/ObjectManagerPersister.php

```diff
@@ -27,35 +27,45 @@ class ObjectManagerPersister implements PersisterInterface
 {
     use IsAServiceTrait;
 
-    private $objectManager;
+    /** @var ObjectManager[]  */
+    private $managers;
 
     /**
-     * @var array|null Values are FQCN of persistable objects
+     * @var ClassMetadata[] Entity metadata, FQCN being the key
      */
-    private $persistableClasses;
+    private $metadata = [];
 
     /**
-     * @var ClassMetadata[] Entity metadata, FQCN being the key
+     * @var string[ObjectManager] Keys are FQCN of persistable objects, Values are managers
```

Comment:
I don't think that's a valid phpdoc type.

Reply:
Yeah, I don't know if it's possible to set the type of keys